lib/Page.js

```diff
@@ -62,6 +66,28 @@ function calculateNewBaseUrl(filePath, root, lookUp) {
   return calculate(filePath, []);
 }
 
+function formatFooter(pageData) {
+  const $ = cheerio.load(pageData);
+  const footers = $('footer');
+  if (footers.length === 0) {
+    return pageData;
+  }
+  // Remove preceding footers
+  footers.slice(0, -1).remove(); // footers.not(':last').remove();
+  // Unwrap last footer
+  const lastFooter = footers.last();
+  const lastFooterParents = lastFooter.parents();
+  if (lastFooterParents.length) {
+    const lastFooterOutermostParent = lastFooterParents.last();
+    lastFooterOutermostParent.after(lastFooter);
+  }
+  // Insert flex div before last footer
+  if (lastFooter.before().attr('id') !== FLEX_DIV_ID) {
```

Comment:
Should be `.prev()` instead of `.before()`. Can test by manually inserting `FLEX_DIV_HTML`.